core/services/fluxmonitor/flux_monitor.go

```diff
@@ -935,11 +935,8 @@ func (p *PollingDeviationChecker) loggerFieldsForAnswerUpdated(log *contracts.Lo
 	}
 }
 
-func (p *PollingDeviationChecker) Consumer() models.LogConsumer {
-	return models.LogConsumer{
-		Type: models.LogConsumerTypeJob,
-		ID:   p.initr.JobSpecID,
-	}
+func (p *PollingDeviationChecker) ConsumerID() *models.ID {
+	return p.initr.JobSpecID
```

Comment:
Since there is only one consumer right now (job spec) then why not simply call it JobSpecID?

We can change it later if we need polymorphism. And its clearer if we never end up adding another consumer. What do you think?

Reply:
yeah I kinda went back and forth on that. I was turned off by doing things like..
```go
logConsumption.ConsumerID() // makes it obvious that this is the thing that consumed the log
logConsumption.JobId() // less obvious what role the job plays in the consumption record. ex is it part of the log?
```

Reply:
@samsondav If you feel like `jobID` is definitely the way to go I'm happy to change. Just want to do whatever is less confusing to the future inheritor of this code.